### DISCUSSION TITLE
Finding specific packages of an organization on the site is difficult

### DISCUSSION THREAD

**Discussion by robopzet** [npm/feedback#895] [Website] [answered]
From the page with packages for an organization it's hard to find a specific package. 

1. text entered in the search box is not limited to our packages
2. text entered in the search box that is included in the name of some of our packages gives an empty listbox 
3. typing the full name, including organization gives an empty listbox
4. because the list of packages on the page has a random order selecting a page to browse to is useless. You never know where a package might appear.

My suggestions:
* make the search box filter the results to include our packages if the entered text appears anywhere in the package name. Sort the results so our packages at the top of the list
* sort the packages on the pages by name so you can page though them.

**Comment by garymoultry** [comment]
https://www.huptechweb.com/link-building-strategy/

**Comment by monishcm** [comment]
Thanks for the suggestion. We will add this to our backlog and contact you if we need more information. Agree this is an issue for orgs with a lot of packages.

**Comment by monishcm** [comment, marked as answer]
By far the easiest way would be to npjs.com/package/<package_name> (if you already know the full package name).

The existing search box searches all the packages and is no filtering ability to filter packages from a specific org.

**Reply by robopzet** [reply]
> By far the easiest way would be to npjs.com/package/<package_name> (if you already know the full package name).

Yeah, that's what I did.
